mw-webapp/src/component/table/Table.tsx

```diff
@@ -1,40 +1,58 @@
-import {useEffect, useState} from "react";
+import {useContext} from "react";
 import {
   flexRender,
-  getCoreRowModel,
-  useReactTable,
+  HeaderGroup,
+  RowModel,
 } from "@tanstack/react-table";
-import {columns} from "src/component/table/columns";
-import {getDayReports} from "src/dataAccessLogic/getDayReports";
 import {DayReport} from "src/model/businessModel/DayReport";
 import styles from "src/component/table/Table.module.scss";
 
 /**
- * Table (need update for split component and logic code)
+ * Context's props table users
+ * @public
  */
-export const Table = () => {
-  const [data, setData] = useState<DayReport[]>([]);
+export interface TableUsers {
```

Comment:
user is business model

Reply:
Right, the user is not here, I renamed it